### PR TITLE
3132 public dashboard fixes

### PR DESCRIPTION
**Pull request by javierarce**
This PR fixes the following things:

- [x] The top blue header should be smaller.
- [x] The top map should be taller in a mobile device.
- [x] The logo in the footer should appear in the first position.
- [x] Remove right white space (this happened because of ```.ContentFooter``` having a defined width, instead of 100%)

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2960/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2976/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2977/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2978/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2979/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2980/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2984/))

**Comment by javierarce** [comment]
@jmedina: can you have a look, please?

File: app/assets/stylesheets/new_common/maps-list.css.scss [hunk 1]
```diff
@@ -47,13 +46,15 @@
     @include flex-direction(row);
     @include justify-content(flex-start);
   }
-  .MapsList-item--fake {
-    display: none;
+  .MapsList .MapsList-item.MapsList-item--fake {
+    display: none!Important;
```
Comment (javierarce) [inline app/assets/stylesheets/new_common/maps-list.css.scss]: I had to do it :dog:

**Comment by javierarce** [comment]
@xavijam: can you have a look, please?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2999/))

**Comment by xavijam** [comment]
What are the changes?

**Comment by javierarce** [comment]
Sorry, I've updated the description with some info.

File: app/assets/stylesheets/new_common/maps-list.css.scss (hunk 1 above)
Comment (xavijam) [inline app/assets/stylesheets/new_common/maps-list.css.scss]: :skull:

**Comment by xavijam** [comment]
:es: tested in staging as well :)

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3011/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3014/))

**Pull request merged**